src/averageAges.js

```diff
@@ -36,8 +47,24 @@
  *
  * @return {number}
  */
-function calculateWomenAverageAge(people, withChildren) {
-  // write code here
+function calculateWomenAverageAge(people, withChildren = 0) {
+  let arrayOfAgeWoman;
+  let avgYear = 0;
+  const parameter = withChildren;
```

Comment:
Unnecessary variable 'parameter'. You could use 'withChildren' directly in your conditional statement.